test/infrastructure/vm-operator/net-operator-networkinterface-status.yaml

```diff
@@ -0,0 +1,138 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Where does this come from? (let's document)

How does this match up to the types we define?

Why do we have this CRD, but not the one from vmxnet3networkinterface_types.go?

Reply:
This is a copy of the CRD in vm-operator with the only addition of the status subresource. As soon as this is fixed in thw vm-operator code base, we can get rid of this patch